composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "wavevision/coding-standard",
   "description": "The Wavevision PHP coding standard.",
-  "type": "library",
+  "type": "phpcodesniffer-standard",
```

Comment:
I am not sure this is correct. This package is not _just_ a codesniffer standard, it includes `.editorconfig`, PhpStorm style etc. Is this custom type needed for some codesniffer autodetection or IDE configuration?

Reply:
1. it descibes the package's purpose
1. `dealerdirect/phpcodesniffer-composer-installer` needs this - providing someone uses that pacakge

Reply:
I agree with @rozsival. Let's keep `library` for now.